lib/manager/batect/extract.spec.ts

```diff
@@ -1,15 +1,11 @@
-import { readFileSync } from 'fs';
 import { id as gitTagDatasource } from '../../datasource/git-tags';
 import { id as dockerVersioning } from '../../versioning/docker';
 import { id as semverVersioning } from '../../versioning/semver';
 import { PackageDependency } from '../common';
 import { getDep } from '../dockerfile/extract';
-import { extractPackageFile } from './extract';
+import { extractAllPackageFiles } from './extract';
 
-const sampleFile = readFileSync(
-  'lib/manager/batect/__fixtures__/batect.yml',
-  'utf8'
-);
+const fixturesDir = 'lib/manager/batect/__fixtures__';
```

Comment:
You'll need to construct these paths with `upath.join`, otherwise the file paths aren't constructed correctly for Windows tests.